net/file_server.ts

```diff
@@ -12,6 +12,8 @@ import {
   Response
 } from "./http.ts";
 import { cwd, DenoError, ErrorKind, args, stat, readDir, open } from "deno";
+import { extname } from "https://deno.land/x/path/index.ts";
```

Comment:
Use relative paths within `deno_std` so that if people link to a specific version they will get a compatible version of paths.

Reply:
Yeah I wasn't sure about that, so went with registry, I'll open up another PR for that